def error_func(i, j, z):
        pass
